low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -221,9 +233,21 @@ def persist_state_data(self, resource, data_path):
         else:
             self.persistent_data[resource] = {
                 f'high_{resource}_usage' : str(self.high_usage[resource]),
+                f'{resource}_usage_time_map' : str(self.usage_time_map[resource])
             }
         store.put(self.persistent_data[resource], PER_DATA_PATH)
 
+    def read_persistent_data(self, data_path):
+        """Read resource data from persistent cache."""
+        PER_DATA_PATH = os.path.join(self.cache_dir_path,
+                            f'{data_path}_{self.node_id}')
+
+        if os.path.isfile(PER_DATA_PATH):
+            persistent_data = store.get(PER_DATA_PATH)
+            return persistent_data
+
+        return None
+

```

Comment:
Not needed. Functions returns None by default.

Reply:
I have added it explicitly because in previous condition has different return value.